Rocket.Chat.Shared/Managers/Launcher/PersistencyCoordinator.swift

```diff
@@ -9,7 +9,8 @@
 import Foundation
 import RealmSwift
 
-struct PersistencyCoordinator: LauncherCoordinator {
+class PersistencyCoordinator: LauncherCoordinator {
```

Comment:
if you are changing it to a `class`, make sure to mark it as final (since there's nothing subclassing it).
That will use direct dispatch, among other compilers optimizations.
What do you think?